core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMetadata.java

```diff
@@ -1202,6 +1202,9 @@ default void dropMaterializedView(ConnectorSession session, SchemaTableName view
 
     /**
      * The method is used by the engine to determine if a materialized view is current with respect to the tables it depends on.
+     *
+     * @throws MaterializedViewNotFoundException when materialized view is not found
+     * @throws TableNotFoundException when underlying storage table of materialized view is not found
```

Comment:
remove `TableNotFoundException when underlying storage table of materialized view is not found`